src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
This loop checks for duplicate values in the input object. However, it does this by comparing each value with every other value, which is not efficient. A more efficient way to check for duplicates would be to add each value to a set as you iterate over the object. If a value is already in the set, then it is a duplicate and you can return null. This would result in a time complexity of O(n).